.github/workflows/branch-build.yaml

```diff
@@ -45,6 +48,9 @@ jobs:
         # See the issue https://github.com/facebook/create-react-app/issues/3657
         run: CI=false npm run build
 
+      - name: Set Docker tag env
```

Comment:
I needed to move this step as it couldn't execute Bash script in non-existing working directory (the repo wasn't yet cloned, so the path didn't exist). Anyway, it makes sense to have it close to the actual Docker build/push logic.